metrics/ci_integrations/test/client/github_actions/mappers/run_conclusion_mapper_test.dart

```diff
@@ -0,0 +1,197 @@
+import 'package:ci_integration/client/github_actions/mappers/run_conclusion_mapper.dart';
+import 'package:ci_integration/client/github_actions/models/run_conclusion.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("RunConclusionMapper", () {
+    const runConclusionMapper = RunConclusionMapper();
+
+    test(
+      ".map() maps the success run conclusion to the RunConclusion.success",
+      () {
+        const conclusion = RunConclusionMapper.success;
+        final runConclusion = runConclusionMapper.map(conclusion);
+
+        const expectedRunConclusion = RunConclusion.success;
```

Comment:
Expected values should come first in the test case body according to the AAA principle.